packages/lesswrong/components/posts/PostsItemComments.tsx

```diff
@@ -42,7 +43,7 @@ const styles = (theme: ThemeType): JssStyles => ({
     color: theme.palette.icon.commentsBubble.noUnread,
   },
   unreadComments: {
-    color: theme.palette.secondary.light,
+    color: forumTypeSetting.get() === 'EAForum' ? theme.palette.primary.main : theme.palette.secondary.light,
```

Comment:
I don't understand how this changed, since it seems like you didn't actually touch this with your PR? Let me know if there's a better way to handle this case.

Reply:
I think it's probably that the value of `theme.palette.secondary.light` changed accidentally. It's used in only two places (the other is the left-border for new comments in `CommentsFrame`). Before the theming system, I believe this was getting filled in by an automagical and opaque procedure by material-UI applying a color-shift to `theme.palette.secondary.main`. I tried to fill in all the colors that were being automagically populated this way, but I may have put the wrong color into this slot.

Reply:
Ah got it - ok I updated our `secondary.light` color and reverted this back